dev/diagrams/sign-out-hooks.md

```diff
@@ -7,7 +7,7 @@
   - [Hooks](#hooks)
     - [useOnSignOutApp](#useonsignoutapp)
     - [useIsAppBooted](#useisappbooted)
-    - [useApiErrorHook](#useapierrorhook)
+    - [useApiError](#useapierror)
```

Comment:
Download the TOC and make it automatically updated
https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc

Reply:
Handled in [fb0157c](https://github.com/ajktown/docs/pull/57/commits/fb0157cc9544f2b1a9b923a6f0f71d43bb4b3e5d)
OMG!!! Such a useful tool!

Reply:
@akanex115 hmm it didnt happen to me but good decision that you manually put in.